tests/contrib/django_util/test_views.py

```diff
@@ -198,17 +199,18 @@ def test_callback_handles_bad_flow_exchange(self, pickle):
             redirect_uri=request.build_absolute_uri('oauth2/oauth2callback'))
 
         session_key = 'google_oauth2_flow_{0}'.format(self.CSRF_TOKEN)
-        self.session[session_key] = pickle.dumps(flow)
+        self.session[session_key] = jsonpickle_mock.encode(flow)
 
         def local_throws(code):
             raise FlowExchangeError('test')
 
         flow.step2_exchange = local_throws
-        pickle.loads.return_value = flow
+        jsonpickle_mock.decode.return_value = flow
 
         request.session = self.session
         response = views.oauth2_callback(request)
         self.assertIsInstance(response, http.HttpResponseBadRequest)
+        self.assertTrue(jsonpickle_mock.encode.called)
```

Comment:
No `assert_called_once_with` is what is desired. It actually specifies which arguments were passed in.